### ISSUE TITLE
Popups doesn't apply the theme of the parent element

### ISSUE THREAD

**Issue by alexdi220** [microsoft/microsoft-ui-xaml#6622]
### Describe the bug

Hi, popups, flyouts, etc. don't listen to the parent's theme changing. For a sample the ComboBox which placed within the StackPanel with RequestedTheme=Dark. ComboBox is black, popup - white. It also reproduces when changing the theme in the runtime. It's an annoying issue when we have popups within our components. We faced it for a long time and found WA - set the RequestedTheme to popup directly. But every time it's sad.

### Steps to reproduce the bug

```xaml
    <StackPanel Background="{ThemeResource AppBarBackgroundThemeBrush}" 
                VerticalAlignment="Center" 
                Spacing="30"
                RequestedTheme="Dark">
        <Button Content="ChangeTheme" Click="Button_Click"/>
        <ComboBox Width="100">
            <ComboBoxItem>Test1</ComboBoxItem>
            <ComboBoxItem>Test2</ComboBoxItem>
        </ComboBox>
    </StackPanel>
```

```csharp
    public sealed partial class MainWindow : Window {
        public MainWindow() {
            this.InitializeComponent();
        }

        private void Button_Click(object sender, RoutedEventArgs e) {
            var element = Content as FrameworkElement;
            element.RequestedTheme = element.ActualTheme == ElementTheme.Light ? ElementTheme.Dark : ElementTheme.Light;
        }
    }
    public class MyCalendar : CalendarView { }
```

### Screenshots

![image](https://user-images.githubusercontent.com/47348109/150144681-5aba5e02-e7eb-4078-a651-42692cd3640a.png)

### NuGet package version

WinUI 3 - Windows App SDK 1.0 (If you're seeing your issue in older previews of WinUI 3, please try this release)

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

May 2021 Update (19043)

**Comment by chausner** [comment]
I've been experiencing the same. It means you cannot really offer a dynamic setting in your app to change the theme, without requiring the user to restart the app afterwards to have the changes take full effect.

**Comment by alexdi220** [comment]
Two weeks have passed. Do you have any plans to fix it or a quality WA instead of my approach?

**Comment by kmgallahan** [comment]
@alexdi220 There are currently [515 issues with the `needs-triage` tag](https://github.com/microsoft/microsoft-ui-xaml/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage). 

On the last community call the lead for WinUI stated he is trying to figure out the problem of how to handle the flood of issues showing up here, and that it would takes weeks before a plan was put together. If/when a plan is put together then the process of actually going through all the backed-up issues would start. That was 2 weeks ago, so it is going to be a while.

**Comment by AndrewFromJapan** [comment]
I posted a workaround I'm using for the moment here #6697. Hope this helps  #until they fix the problem.

**Comment by bpulliam** [comment]
Using the sample above and WinUI Gallery, I am seeing this as fixed in 1.2.
